.github/ISSUE_TEMPLATE/question.yaml

```diff
@@ -0,0 +1,51 @@
+name: Question
+description: Ask a question
+title: 'question: '
+labels: [question]
+
+assignees: [rycee, berbiche]
+
+body:
+- type: textarea
+  id: question
+  attributes:
+    label: Question
+  validations:
+    required: true
+- type: textarea
+  id: maintainers
+  attributes:
+    label: Maintainer CC
+    description: |
+      Please @ people who are in the `meta.maintainers` list of the module in question.
+      If in doubt, check `git blame` for whoever last touched something.
+  validations:
+    required: true
+
+- type: markdown
+  attributes:
+    value: |
+      Information about how you use Nix and Home Manager and your system that runs them.
```

Comment:
Maybe the markdown should be a title?

![image](https://user-images.githubusercontent.com/20448408/154828447-4c344c3e-56d2-49d4-bcee-33125d7958d5.png)

Reply:
Correction: can this sentence be moved before "Please run ..."?

Reply:
@berbiche Which sentence? Should I move the `Information about...` part right before the `Please run...` part?

Reply:
The one above "System": "Information about how you use ..."